spec/spec_helper.rb

```diff
@@ -14,13 +14,6 @@
 require 'vcr'
 require 'cgi'
 
-# FIXME: This bypasses the lookup of Javascript dependencies in tests until they can be found and uncoupled
-class << ActionController::Base.helpers
-  def image_path(path, options = {})
-    path
-  end
-end
-
```

Comment:
This was so we didn't need `bower` dependencies installed when running specs.

So.. trying removing ui's `vendor/assets/bower_components` (or simulate github 404ing) and running the specs to see what happens...

That said.. I don't see any `image_path` call in manageiq/ so it should indeed not longer be needed here, unless we still call some UI bits in the specs :).